tests/test_custom_ca.py

```diff
@@ -0,0 +1,176 @@
+import asyncio
+import pathlib
+import ssl
+import typing
+from dataclasses import dataclass
+from tempfile import TemporaryDirectory
+
+import pytest
+import pytest_asyncio
+import urllib3
+from aiohttp import ClientSession, web
+
+import truststore
+
+pytestmark = pytest.mark.asyncio
```

Comment:
```suggestion
```

This appears to be unused

Reply:
ok was able to remove it again after touching up the other decorators

Reply:
@adriangb oh sorry, that's right, I forgot how `pytestmark` works. The way you have it currently is fine.